src/content/keyAutoAdd/lib/keyAutoAdd.ts

```diff
@@ -45,6 +49,78 @@ export class KeyAutoAdd {
     );
   }
 
+  private setNotificationSize(size: 'notification' | 'fullscreen' | 'hidden') {
+    let styles: Partial<CSSStyleDeclaration>;
+    const defaultStyles: Partial<CSSStyleDeclaration> = {
+      outline: 'none',
+      border: 'none',
+      zIndex: '9999',
+      position: 'fixed',
+      top: '0',
+      left: '0',
+    };
+
+    if (size === 'notification') {
+      styles = {
+        width: '22rem',
+        height: '8rem',
+        position: 'fixed',
+        top: '1rem',
+        right: '1rem',
+        left: 'initial',
+        boxShadow: 'rgba(0, 0, 0, 0.1) 0px 0px 6px 3px',
+        borderRadius: '0.5rem',
+      };
+    } else if (size === 'fullscreen') {
+      styles = {
+        width: '100vw',
+        height: '100vh',
+      };
+    } else {
+      styles = {
+        width: '0',
+        height: '0',
+        position: 'absolute',
+      };
+    }
+
+    this.ui.style.cssText = '';
```

Comment:
Why are setting `cssText` to an empty string?

Reply:
To reset any inline styles set by previous calls to this function. Basically like `innerHTML = ''; appendChild()...`